### DISCUSSION TITLE
Cancel my membership

### DISCUSSION THREAD

**Discussion by olimjon1606** [planetscale/discussion#606] [FAQs] [answered]
i was trying to create free database but after adding the card it's saying pending 30$ will be charged next month. How can i cancel my plan and remove my card?

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @olimjon1606,

For billing questions, it's best to contact support directly: https://support.planetscale.com/hc/en-us. Please include your account information. They'll be able to help you best.

**Comment by Khosbayar22** [comment]
I'm experiencing a similar issue. I had been utilizing a free plan, but unexpectedly $40 was deducted from my card. I find the process for canceling the plan quite unclear. Could you please advise me on how to obtain a refund for the unused portion of my database subscription?

**Reply by Khosbayar22** [reply]
Give a clear answer please !

**Reply by mscoutermarsh** [reply]
For billing questions, it's best to contact support directly: https://support.planetscale.com/hc/en-us. Please include your account information. They'll be able to help you best.
